src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -125,6 +127,7 @@ static string LocalAddress(IBuilder builder)
             return builder.Build<ReadOnlySettings>().LocalAddress();
         }
 
+        /// Not sure about this one
```

Comment:
The persistence should provide factory methods just like @SzymonPobiega recently changed the `TransportDefinition` to require. If we had this the feature would do:

`settings.Get<PersistenceDefinition>().CreateSubscriptionStorage()`